main/FatesHistoryInterfaceMod.F90

```diff
@@ -4540,12 +4540,12 @@ subroutine define_history_vars(this, initialize_variables)
           upfreq=1, ivar=ivar, initialize=initialize_variables, index = ih_m5_si_scpf )
 
     call this%set_history_var(vname='CROWNFIREMORT_SCPF', units = 'N/ha/yr',          &
-          long='crown fire mortality by pft/size',use_default='inactive', &
+          long='crown fire mortality by pft/size',use_default='active', &
```

Comment:
```suggestion
          long='crown scorch fire mortality by pft/size',use_default='active', &
```

Reply:
@jkshuman in [this suggestion](https://github.com/NGEET/fates/pull/584#discussion_r349351926) you changed the variable name from CROWNFIREMORT to CROWNSCORCHMORT.

If that is correct, then should your `long` suggestion replace "crown fire mortality" with "crown scorch mortality"?

Reply:
...instead of "crown scorch fire mortality"